gr-blocks/lib/correctiq_impl.cc

```diff
@@ -0,0 +1,63 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "correctiq_impl.h"
+#include <gnuradio/io_signature.h>
+
+namespace gr {
+namespace blocks {
+
+correctiq::sptr correctiq::make()
+{
+    return gnuradio::get_initial_sptr(new correctiq_impl());
+}
+
+/*
+ * The private constructor
+ */
+correctiq_impl::correctiq_impl()
+    : gr::sync_block("correctiq",
+                     gr::io_signature::make(1, 1, sizeof(gr_complex)),
+                     gr::io_signature::make(1, 1, sizeof(gr_complex)))
+{
+    avg_real = 0.0;
```

Comment:
these should be moved to either the instantiation list (just above, right after the `gr::sync_block` or into the `_impl.h` class itself.

Reply:
oh ... you've already set these to 0.0 in the `_impl.h` ... so these lines here aren't required at all.